Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/PatientDiscoveryDeferredResponseAuditLog.properties

```diff
@@ -51,7 +40,7 @@ ActiveParticipant_Dest.RoleIDCode.@code=110152
 ActiveParticipant_Dest.RoleIDCode.@codeSystemName=DCM
 ActiveParticipant_Dest.RoleIDCode.@displayName=Destination
 
-AuditSourceIdentification.@AuditEnterpriseSiteID=Gateway 1
+AuditSourceIdentification.@AuditEnterpriseSiteID=DoD
 AuditSourceIdentification.@AuditSourceID=urn:oid:1.1
 AuditSourceIdentification.@AuditSourceTypeCode=1010
 
```

Comment:
Yes, During our discussions we thought of having at g0, g1 suite level. But not always we run complete suite to execute test, sometimes we may end up running a particular testcase ( a particular service ). This is the reason, later decided to have setup, teardown scripts at test case level.